src/objects/deck/deck.scss

```diff
@@ -2,7 +2,7 @@
 @use '../../compiled/tokens/scss/size';
 @use '../../mixins/fluid';
 @use '../../mixins/media-query';
-@use '../container/container.scss';
+@use '../container/container';
```

Comment:
The changes in this file are just formatting changes I noticed were needed while copying some code.

Reply:
I realize this line isn't new, but is this resulting in the container CSS being output an additional time? If so, can we use design tokens to share whatever values we're harvesting from this file instead of including the whole file?